src/types.ts

```diff
@@ -62,10 +70,13 @@ export type SDKMessageIds = keyof typeof SDK_MESSAGES;
 
 export interface InterfaceMessageToPayload {
   [INTERFACE_MESSAGES.ON_SAFE_INFO]: SafeInfo;
+  [INTERFACE_MESSAGES.TRANSACTION_CONFIRMED]: {
+    safeTxHash: string;
+  };
 }
 
 export type SentSDKMessage<T extends SDKMessageIds> = {
   messageId: T;
-  requestId: number | string;
+  requestId: RequestId;
```

Comment:
Not sure if we should make this optional .... cannot think of a good reason ... just a feeling :D

Reply:
But this type is for the outgoing message, it will always have a request id. Either a user can provide it or it will be auto-generated